docs/source/upload_dataset.mdx

```diff
@@ -125,8 +125,8 @@ Congratulations, you've completed the tutorials! 🥳
 
 From here, you can go on to:
 
-- Learn more about how to use 🤗 Datasets other functions to [process your dataset](/process).
-- [Stream large datasets](/stream) and avoid waiting for the entire dataset to download.
-- [Write a dataset loading script](/dataset_script) and share your dataset with the community.
+- Learn more about how to use 🤗 Datasets other functions to [process your dataset](https://huggingface.co/docs/datasets/process).
+- [Stream large datasets](https://huggingface.co/docs/datasets/stream) and avoid waiting for the entire dataset to download.
+- [Write a dataset loading script](https://huggingface.co/docs/datasets/dataset_script) and share your dataset with the community.
```

Comment:
Thanks !

A relative import works as well ;)
```suggestion
- Learn more about how to use 🤗 Datasets other functions to [process your dataset](process).
- [Stream large datasets](stream) and avoid waiting for the entire dataset to download.
- [Write a dataset loading script](dataset_script) and share your dataset with the community.
```

And this way users can navigate through the documentation at the same version (using an absolute URL links to the latest version)